fuse/data/utils/samplers.py

```diff
@@ -61,7 +61,9 @@ def __init__(
                                         In mode 'exact' the weights should be integers that sums up to batch size.
                                         In mode 'approx' the weights should be floats that sums up to ~1
                                         If not specified, equal number of samples from each class will be used.
-        :param num_batches: optional - if set will force num_batches, otherwise num_batches will be automatically to go over each sample at least once (exactly or approximately).
+        :param num_batches: optional - if set to an int will force num_batches,
```

Comment:
Cool!
worth adding to the function signature that `str` is also a valid type for `num_batches`

Reply:
Thanks.
I will add it in my next PR though.